apollo-compiler/src/test/graphql/com/example/unique_type_name/HeroDetailQueryExpected.java

```diff
@@ -61,9 +61,9 @@ public String queryDocument() {
   }
 
   public interface Data extends Operation.Data {
-    @Nullable Hero hero();
+    @Nullable HeroDetailQuery$ heroDetailQuery();
```

Comment:
And just for clarity 5 would be Query$$$$$ so just as example imagine you see following in logs... 
NullPointerExxeption: Foo$$$$ is null 

Will you be able to tell without dragging your finger in screen whether that was 4 or 5 dollar signs. 
I don't feel strongly about it but can see it leading to confusing traces. As example rxjava does Action0  -  Action9